apps/src/templates/projects/FeaturedProjectsTable.story.jsx

```diff
@@ -6,29 +6,21 @@ import {
 } from './generateFakeProjects';
 import {featuredProjectTableTypes} from './projectConstants';
 
-export default storybook => {
-  return storybook
-    .storiesOf('Projects/FeaturedProjectsTable', module)
-    .addStoryTable([
-      {
-        name: 'Featured Project Table - current',
-        description: 'Table of currently featured projects projects',
-        story: () => (
-          <FeaturedProjectsTable
-            projectList={stubFakeFeaturedProjectData}
-            tableVersion={featuredProjectTableTypes.current}
-          />
-        )
-      },
-      {
-        name: 'Featured Project Table - archive',
-        description: 'Table of projects that have previously been featured',
-        story: () => (
-          <FeaturedProjectsTable
-            projectList={stubFakeUnfeaturedProjectData}
-            tableVersion={featuredProjectTableTypes.archived}
-          />
-        )
-      }
-    ]);
+const Template = args => <FeaturedProjectsTable {...args} />;
+
+export const FeaturedProjectTableCurrent = Template.bind({});
+FeaturedProjectTableCurrent.args = {
+  projectList: stubFakeFeaturedProjectData,
+  tableVersion: featuredProjectTableTypes.current
+};
+
+export const FeaturedProjectTableArchive = Template.bind({});
+FeaturedProjectTableArchive.args = {
+  projectList: stubFakeUnfeaturedProjectData,
+  tableVersion: featuredProjectTableTypes.archived
+};
+
+export default {
+  title: 'Featured Projects Table',
```

Comment:
nit: for each default export, we want the title to match the component name (no spaces)